app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectPresenter.kt

```diff
@@ -92,6 +97,10 @@ class DeviceConnectPresenter(deviceActivity: DeviceActivity, manager: WifiManage
         checkPermissions = b
```

Comment:
Create issue to remove single character variable names

Reply:
https://github.com/fossasia/susi_android/issues/1688 Made